test/functional/test_framework/key.py

```diff
@@ -181,6 +181,8 @@ def sign_ecdsa(self, msg, low_s=True, rfc6979=False):
         s = (pow(k, -1, ORDER) * (z + self.secret * r)) % ORDER
         if low_s and s > secp256k1.GE.ORDER_HALF:
```

Comment:
Could use `elif`, but as the effect is the same, better to roll this into one `if`, i think:
```python
if (low_s and s > secp256k1.GE.ORDER_HALF) or (not low_s and s < secp256k1.GE.ORDER_HALF):
    s = ORDER - s
```

Reply:
Right it's fine, it's documented well enough now, if anyone needs that functionality in the future they can add it then.